103_lineshape_broadening.ipynb

```diff
@@ -0,0 +1,874 @@
+    "\n",
+    "In reality the collisional broadneing coefficients also depends on the collisional partner. \n",
+    "\n",
+    "At first order it is the sum of the partial fractions of all components of the gas \n",
+    "mixture, multiplied by the respective broadening coefficient. \n",
+    "\n",
+    "Example: for a line of CO in a mixture of CO2, H2O, N2, there is \"self-broadening\" \n",
+    "with CO (coefficient :math:\\\\gamma_{CO, CO)) and broadening with CO2, H2O and N2 \n",
+    "(coefficients :math:\\\\gamma_{CO_2, CO), etc.)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "ef95684b",
+   "metadata": {},
+   "source": [
+    "# $ hwhm\\_total = \\sum_i={CO, CO_2,H_2O, N_2} \\gamma_{i, CO} \\cdot P\\_i $"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "bbf18c81",
+   "metadata": {},
+   "source": [
+    "By default in HITRAN and RADIS, only two components are allowed : the molecule of \n",
+    "the transition, and a single diluent, by default the air. The partial pressure \n",
+    "of the emitting molecule is :math:`x \\cdot P`, the rest of the diluent is \n",
+    ":math:`(1-x) \\cdot P`. We get : \n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "d0bd90f7",
+   "metadata": {},
+   "source": [
+    "# $ hwhm\\_collisions=\\gamma_{self} P x + \\gamma_{air} P (1-x) $"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "5c6ebf7e",
+   "metadata": {},
+   "source": [
+    "The broadening did not change much, because the broadening coefficients of CO \n",
+    "collisionned by CO or by air are quite similar. It can be very different \n",
+    "when the collisional partner is a larger molecule, like CO2 or H2O : differences of more than 100% can be obtained. \n",
+    "The formula above is actually an approximation.\n",
+    "\n",
+    "In reality, the collisional broadening also depend on the temperature. The HITRAN database includes\n",
+    "temperature correction coefficients for self-broadening ``Tdpsel`` and air broadening ``Tdpair``: "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "3964e860",
+   "metadata": {},
+   "source": [
+    "# $hwhm\\_{collisions}={\\left(\\frac{T_{ref}}{T_{gas}}\\right)}^{n_{air}} \\gamma_{air} P \\left(1-x\\right)+{\\left(\\frac{T_{ref}}{T_{gas}}\\right)}^{n_{self}} \\gamma_{self} P x $\n",
+    "\n",
+    "This is the equation implemented in RADIS. We obtain it with"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 10,
+   "id": "d1913575",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "pressure broadening at 1 atm with 30% CO, 70% air, and T-dependance :  0.04151353450608599 cm-1\n"
+     ]
+    }
+   ],
+   "source": [
+    "# :py:func:`~radis.lbl.broadening.pressure_broadening_HWHM`\n",
```

Comment:
add reference to Radis function in markdown : 

https://radis.readthedocs.io/en/latest/source/radis.lbl.broadening.html#radis.lbl.broadening.pressure_broadening_HWHM